Makefile

```diff
@@ -1,10 +1,16 @@
 #
 # Makefile - `gocat` (https://github.com/tuladhar/gocat)
 # 
-PROGRAM_NAME:=gocat
+PROGRAM_NAME=gocat
+INSTALL_PATH=/usr/local/bin
 
-build:
+install:
 	go build -o $(PROGRAM_NAME) src/main.go
+	@sudo install $(PROGRAM_NAME) $(INSTALL_PATH)/$(PROGRAM_NAME) 
+	echo "$(PROGRAM_NAME): installed at $(INSTALL_PATH)" 
 
-cleanup:
```

Comment:
Similarly, let's have `uninstall` target separately rather than mixing both cleanup and uninstall.